scripts/obtain/dotnet-install.sh

```diff
@@ -173,10 +182,10 @@ get_current_os_name() {
         local linux_platform_name
         linux_platform_name="$(get_linux_platform_name)" || { echo "linux" && return 0 ; }
 
-        if [[ $linux_platform_name == "rhel.6" ]]; then
+        if [ "$linux_platform_name" = "rhel.6" ]; then
```

Comment:
What is the purpose of this change?

Reply:
It was to improve readability of this `function get_current_os_name()`; rest of `if` conditions in this function (and most in this file) are using POSIX shell's syntax for comparing strings (single square brackets with proper quotes).